ncm-accounts/src/main/pan/components/accounts/schema.pan

```diff
@@ -6,6 +6,22 @@ declaration template components/accounts/schema;
 
 include { 'quattor/schema' };
 
+function has_unique_attr = {
+    values = ARGV[0];
+    attr = ARGV[1];
+    cattrs = list();
+
+    foreach(ni;el;values) {
+        if(index(el[attr], cattrs) != -1) {
+            error(format("Duplicate attr %s : %s ", attr, el[attr]));
+            return(false);
+        } else {
+            append(cattrs, el[attr]);
+        };
+    };
+    return(true);
```

Comment:
IIRC, it's more efficient to just say "true" here and not explicitly call return.